encoder/vaapiencoder_base.cpp

```diff
@@ -233,6 +250,27 @@ YamiStatus VaapiEncoderBase::setParameters(VideoParamConfigType type, Yami_PTR v
             ret = YAMI_INVALID_PARAM;
         }
         break;
+        case VideoParamsTypeQualityLevel: {
+            VideoParamsQualityLevel* videoQualityLevel = (VideoParamsQualityLevel*)videoEncParams;
+            if (videoQualityLevel->size == sizeof(VideoParamsQualityLevel)) {
+                if (videoQualityLevel->level != m_videoParamQualityLevel.level) {
+                    PARAMETER_ASSIGN(m_videoParamQualityLevel, *videoQualityLevel);
```

Comment:
this is duplicate code with L241, could we use a function for this. since as 
ADJUST_TO_RAGE(value, min, max, promp)
So you can call it like this ADJUST_TO_RANGE(m_videoParamQualityLevel.level, VIDEO_PARAMS_QUALITYLEVEL_MIN, VIDEO_PARAMS_QUALITYLEVEL_MAX, "quality level");